pkg/daemon/ceph/osd/volume.go

```diff
@@ -613,20 +633,49 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 			"json",
 		}...)
 
-		cvOut, err := context.Executor.ExecuteCommandWithCombinedOutput(baseCommand, reportArgs...)
-		if err != nil {
-			return errors.Wrapf(err, "failed ceph-volume json report: %s", cvOut) // fail return here as validation provided by ceph-volume
-		}
+		cephVersion := a.clusterInfo.CephVersion
 
-		logger.Debugf("ceph-volume report: %+v", cvOut)
+		// ceph version v14.2.13 ~ v14.2.16 changes output of `lvm batch --prepare --report`
+		// use previous logic if ceph version does not fall into this range
+		if !isNewStyledLvmBatch(cephVersion) {
```

Comment:
minor suggestion instead of declaring a separate variable.
```suggestion
		if !isNewStyledLvmBatch(a.clusterInfo.CephVersion) {
```